src/page/i-frame.page.ts

```diff
@@ -2,9 +2,11 @@ import { $, browser, ElementFinder, promise } from 'protractor';
 
 export class IFramePage {
   private iFrame1: ElementFinder;
+  private title: ElementFinder;
 
   constructor() {
     this.iFrame1 = $('#frame1');
+    this.title = $('');
```

Comment:
why is this selector empty?

Reply:
That was from the previous point. I started to do it but for the change of URL that point can't be completed.

Reply:
Should I deleted it?